<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_filter">
     <property name="text">
      <string>&amp;Filter</string>
     </property>
     <property name="buddy">
      <cstring>edit_filter</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edit_filter"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="OphydDeviceTableView" name="device_table_view">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OphydDeviceTableView</class>
   <extends>QTableView</extends>
   <header>atef.widgets.ophyd</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
